src/main/java/fr/gramlab/frames/GramlabFrame.java

```diff
@@ -84,14 +84,7 @@
 import fr.umlv.unitex.config.ConfigManager;
 import fr.umlv.unitex.files.FileUtil;
 import fr.umlv.unitex.files.PersonalFileFilter;
-import fr.umlv.unitex.frames.AboutDialog;
-import fr.umlv.unitex.frames.DelaFrame;
-import fr.umlv.unitex.frames.FileEditionTextFrame;
-import fr.umlv.unitex.frames.GraphFrame;
-import fr.umlv.unitex.frames.InternalFrameManager;
-import fr.umlv.unitex.frames.MenuAdapter;
-import fr.umlv.unitex.frames.TextAutomatonFrame;
-import fr.umlv.unitex.frames.UnitexFrame;
+import fr.umlv.unitex.frames.*;
```

Comment:
Do not use [wildcard imports](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports).